### PR TITLE
Minor update - creat nodes from SpatialLinesNetwork

### PR DESCRIPTION
**Pull request by Robinlovelace**
Thought @richardellison? `sln2points()` seems to work. We should probably write something on what these nodes represent - all points on the network where lines intersect, right?

Any feedback welcome.

### CODE REVIEW

**Comment by richardellison** [comment]
At the moment it is all lines where a coordinate that defines the line intersects with another line. I'm trying to see if I can make it any intersection (as an option) for data where the roads have not be created using intersecting nodes but that this is the desired functionality (it isn't always). This is to help with OSM data in particular where roads do not necessarily intersect at a node even if there is a physical intersection.

**Comment by Robinlovelace** [comment]
OK - do you want to add it to this PR or start a new one?

**Comment by richardellison** [comment]
I'll add it to a new one I think.

File: man/sln2points.Rd [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+\description{
+Generate spatial points representing nodes on a SpatialLinesNetwork
+}
+\examples{
+data(routes_fast)
+rnet <- overline(sldf = routes_fast, attrib = "length")
+SLN <- SpatialLinesNetwork(rnet)
+(sln_nodes = sln2points(SLN))
+plot(SLN)
+plot(sln_nodes, add = T)
```
Comment (richardellison) [inline man/sln2points.Rd]: The build is failing here because it should be:

```{r}
plot(sln_nodes, add = TRUE)
```

**Pull request merged**